### ISSUE TITLE
Cannot transcribe text into prompt using macos accessibility tools

### ISSUE THREAD

**Issue by savanthongvanh** [openai/codex#1851]
### What version of Codex is running?

codex-cli 0.11.0

### Which model were you using?

codex-mini-latest

### What platform is your computer?

Darwin 24.3.0 arm64 arm

### What steps can reproduce the bug?

- Configure dication per instructions (i use ctrl key twice as shortcut)
https://support.apple.com/guide/mac-help/use-dictation-mh40584/mac

# works
- open terminal
- hit control key twice
- say ls
- see macos typing "ls"

# doesnt work
- open terminal
- start codex
```codex```
- hit control key twice
- say "potato"
- observe codex doing nothing

Seems like the dictation feature should work as this works.

it does work if you start the dictation before you start codex

### What is the expected behavior?

- open terminal
- start codex
```codex```
- hit control key twice
- say "potato"
- observe potato being typed

### What do you see instead?

i see codex doing nothing

**Comment by tibo-openai** [comment]
https://github.com/user-attachments/assets/bd253694-fc8e-41d6-b701-18361690a1ee

I think this is fixed in the latest releases. Thank you for making me say potato to my computer. Have a nice day and going to mark this as resolved.
